### PR TITLE
Add featureCounts tool

### PR DESCRIPTION
**Pull request by upendrak**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by upendrak** [comment]
Why does the Travis build failed this time? The `conda-build` command worked great

File: recipes/featurecounts/1.5.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package:
+  name: featurecounts
+  version: "v1.5.1"
+
+source:
+  fn: subread-1.5.1-source.tar.gz
+  url: https://sourceforge.net/projects/subread/files/subread-1.5.1/subread-1.5.1-source.tar.gz/download
```
Comment (bgruening) [inline recipes/featurecounts/1.5.1/meta.yaml]: the download at the end can be removed.

File: recipes/featurecounts/1.5.1/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  md5: 3eb9a3ff0fc13401a1d76681549494f5
+
```
Comment (bgruening) [inline recipes/featurecounts/1.5.1/meta.yaml]: you don't have any build and runtime dependencies defined, this does not work

**Comment by bgruening** [comment]
The featurecounts tool is called subread and the recipe can be found here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/subread/meta.yaml

File: recipes/featurecounts/1.5.1/meta.yaml (hunk 1 above)
Comment (upendrak) [inline recipes/featurecounts/1.5.1/meta.yaml]: Ok got it

File: recipes/featurecounts/1.5.1/meta.yaml (hunk 2 above)
Comment (upendrak) [inline recipes/featurecounts/1.5.1/meta.yaml]: Ok i will add it now..

**Comment by bgruening** [comment]
@upendrak if you build your package locally with `conda build` you local environment can leak into the build environment, e.g. python and perl is installed and can be used. Here on travis is will not work, as we build in a build environment where no perl/python... is installed, so you need to define it.

**Comment by upendrak** [comment]
Oh i see. Shall i fix it and then push the changes or you don't want me to do that since the subread is already present (that is old version though)

**Comment by bgruening** [comment]
This was just to let you know what went wrong. I think we should close this PR as the package already exists.

**Comment by upendrak** [comment]
Ok got it..

**Pull request closed without merging**